plugins/juju/pyparts/units.py

```diff
@@ -1,10 +1,35 @@
 from core.plugins.juju import JujuChecksBase
+from core.plugins.storage.bcache import BcacheBase
+from core.issues import (
+    issue_types,
+    issue_utils,
+)
 
 YAML_PRIORITY = 2
 
 
 class JujuUnitChecks(JujuChecksBase):
 
+    def bcache_tuning_unit(self):
+        """
+        Check if bcache-tuning charm is deployed for Ceph with bcache.
+        """
```

Comment:
Hi Pon, since this is a bcache check I feel it really belongs in plugins/storage/pyparts/bcache.py.

Reply:
Thanks. Done now.